Documentation/ceph-upgrade.md

```diff
@@ -476,15 +486,17 @@ located.
 ```yaml
   env:
     - name: ROOK_CSI_CEPH_IMAGE
-        value: "quay.io/cephcsi/cephcsi:v1.2.2"
+        value: "quay.io/cephcsi/cephcsi:v2.0.0"
     - name: ROOK_CSI_REGISTRAR_IMAGE
-        value: "quay.io/k8scsi/csi-node-driver-registrar:v1.1.0"
+        value: "quay.io/k8scsi/csi-node-driver-registrar:v1.2.0"
     - name: ROOK_CSI_PROVISIONER_IMAGE
         value: "quay.io/k8scsi/csi-provisioner:v1.4.0"
     - name: ROOK_CSI_SNAPSHOTTER_IMAGE
         value: "quay.io/k8scsi/csi-snapshotter:v1.2.2"
     - name: ROOK_CSI_ATTACHER_IMAGE
-        value: "quay.io/k8scsi/csi-attacher:v1.2.0"
+        value: "quay.io/k8scsi/csi-attacher:v2.1.0"
```

Comment:
To make sure I'm clear on our discussion yesterday, to support the different version of the csi attacher, you are planning to add the version check in the backport PR to support the different params, right?

Reply:
Yes @travisn will do it during backport